releaseNotes/Microsoft.Featuremanagement.md

```diff
@@ -2,6 +2,64 @@
 
 [Source code][source_code] | [Package (NuGet)][package] | [Samples][samples] | [Product documentation][docs]
 
+## 4.0.0 - September 24, 2024
+
+### Enhancements
+
+#### IVariantFeatureManager
+
+Added `IVariantFeatureManager` which is the successor of the existing `IFeatureManager`. It continues to offer the functions of `IFeatureManager`, but offers the new `GetVariantAsync` methods and offers `CancellationToken` on all methods.
+
+#### Variants
+
+```csharp
+Variant variant = await featureManager.GetVariantAsync(MyFeatureFlags.HelpText);
+
+model.Text = variant.Configuration.Value;
+```
+
+This library now supports feature flags with variants. Variants are a tool that can be used to surface different variations of a feature to different segments of an audience. Previously, flags were limited to boolean values, as they are either enabled or disabled. Variants have dynamic values. There can be more than two and they can be any valid JSON value- like a string, int, or complex object.
+
+For more details on Variants, see [here](https://learn.microsoft.com/en-us/azure/azure-app-configuration/feature-management-dotnet-reference#variants).
+
+#### Variant Service Provider
+
+```csharp
+IVariantServiceProvider<IAlgorithm> algorithmServiceProvider;
+...
+
+IAlgorithm forecastAlgorithm = await algorithmServiceProvider.GetServiceAsync(cancellationToken);
+```
```

Comment:
Honestly speaking, without reading the document, these two lines don't mean much. I would just drop it and let people read the document.